apps/src/code-studio/initializeCodeMirror.js

```diff
@@ -41,6 +44,26 @@ module.exports = function (target, mode, callback, attachments) {
   if (mode === 'markdown') {
     backdrop = mode;
     mode = 'spell-checker';
+
+    // In markdown mode, look for a preview element (found by just appending
+    // _preview to the target id), if it exists extend our callback to update
+    // the preview element with the markdown contents
+    const previewElement = document.getElementById(target + '_preview');
```

Comment:
I don't *love* this approach of requiring the preview element to have the same id + "_preview", but it seemed better than having to pass in another param.